Update schedule-replicas-by-topology-labels.md (#7432)
pingcap/docs: schedule-replicas-by-topology-labels.md

## Configure `labels` based on the cluster topology

> **Note:**
>

> > Location labels are defined as Strings in PD with no restricted default or pre-configured values. This offers customizable label configuration in label names other than `zone`, `rack` or `host` which are used as demonstration in below sections.

Reviewer: ```suggestion
> Location labels are defined as strings in PD with no restricted default or pre-configured values. This offers customizable label configuration in label names other than `zone`, `rack` or `host` which are used as demonstration in below sections.
```

Reviewer 2: should "Location labels" be changed to `location-labels`? Also [here](https://github.com/pingcap/docs/blob/7ba53b86478e443ddd76e5e88df8ff3bbc2e6f68/schedule-replicas-by-topology-labels.md?plain=1#L63) has already mentioned the similar concept in the same doc.

> > Location labels are defined as strings in PD with no restricted default or pre-configured values. This offers customizable label configuration in label names other than `zone`, `rack` or `host` which are used as demonstration in below sections.

Reviewer 3: ```suggestion
> Location labels are defined as strings in PD with no restricted default or pre-configured values. This offers customizable label configuration in label names other than `zone`, `rack`, or `host` which are used as demonstrations in the below sections.
```

According to the description above, the label can be any key-value pair used to describe TiKV attributes. But PD cannot identify the location-related labels and the layer relationship of these labels. Therefore, you need to make the following configuration for PD to understand the TiKV node topology.

> Defined as an array of strings, `location-labels` is the configuration for PD. Each item of this configuration corresponds to the key of TiKV `labels`. Besides, the sequence of each key represents the layer relationship of different labels (the layer levels decrease from left to right).

Reviewer 4: I prefer using `isolation level` rather than `layer level`.

Reviewer 3: > I prefer using `isolation level` rather than `layer level`.

have fixed, PTAL, thanks!

> Defined as an array of strings, `location-labels` is the configuration for PD. Each item of this configuration corresponds to the key of TiKV `labels`. Besides, the sequence of each key represents the layer relationship of different labels (the layer levels decrease from left to right).

Reviewer 3: ```suggestion
Defined as an array of strings, `location-labels` is the configuration for PD. Each item of this configuration corresponds to the key of TiKV `labels`. Besides, the sequence of each key represents the layer relationship of different labels (the isolation levels decrease from left to right).
```

You can customize the value of `location-labels`, such as `zone`, `rack`, or `host`, because the configuration does not have default values. Also, this configuration has **no** restriction in the number of label levels (not mandatory for 3 levels) as long as they match with TiKV server labels.

> **Note:**
>

> > You must configure `location-labels` for PD and `labels` for TiKV at the same time for the configurations to take effect. Otherwise, PD does not perform scheduling according to the topology.

Reviewer 3: ```suggestion
> To make configurations take effect, you must configure `location-labels` for PD and `labels` for TiKV at the same time. Otherwise, PD does not perform scheduling according to the topology.
```
